graphene_django/compat.py

```diff
@@ -19,4 +22,16 @@ def __init__(self, *args, **kwargs):
         RangeField,
     )
 except ImportError:
-    IntegerRangeField, ArrayField, HStoreField, RangeField = (MissingType,) * 4
+    IntegerRangeField, HStoreField, RangeField = (MissingType,) * 3
+
+    # For unit tests we fake ArrayField using JSONFields
+    if os.path.basename(sys.argv[0]) in ["pytest", "py.test"]:
```

Comment:
nitpick: this would not work if someone calls `python -m pytest` since `sys.argv[0]` would be `../pytest/__main__.py`. This would be a bit more robust

```python
from pathlib import PurePath

if any(
    PurePath(sys.argv[0]).match(p)
    for p in [
        "**/pytest",
        "**/py.test",
        "**/pytest/__main__.py",
    ]
):
```